### PR TITLE
update GSS-TSIG docs

### PR DESCRIPTION
**Pull request by mmoll**
purely taken from #388

### CODE REVIEW

File: _includes/manuals/1.10/4.3.5.3_gsstsig.md [hunk 1]
```diff
@@ -32,8 +33,21 @@ Once the keytab file has been created, test it using kinit:
...
 
 Store the keytab at `/etc/foreman-proxy/dns.keytab`, ensure permissions are 0600 and the owner is `foreman-proxy`.
+If you are using SELinux, do not forget to update the file context.
 
-The DNS zone `Dynamic Updates` option on the DNS zones can now be set to `Secure Only`. Now follow the steps below under `Proxy Configuration`.
+Now update the configuration in /etc/foreman-proxy/settings.d/dns.yml for the smart proxy:
+
+    :enabled: true
+    :dns_provider: nsupdate_gss
+    :dns_key: false
```
Comment (domcleal) [inline _includes/manuals/1.10/4.3.5.3_gsstsig.md]: These settings have changed in 1.10, they're now split into dns_nsupdate_gss.yml etc.

File: _includes/manuals/1.10/4.3.5.3_gsstsig.md [hunk 2]
```diff
@@ -53,7 +67,7 @@ Note the `\047` is written verbatim, and don't forget the semicolon.  ACLs shoul
 
 #### Proxy configuration
 
-Next, update the proxy DNS configuration file (`/etc/foreman-proxy/settings.d/dns.yml`) with the following setting:
+Update the proxy DNS configuration file (`/etc/foreman-proxy/settings.d/dns.yml`) with the following setting:
```
Comment (domcleal) [inline _includes/manuals/1.10/4.3.5.3_gsstsig.md:60]: Here too actually.

Comment (domcleal) [inline _includes/manuals/1.10/4.3.5.3_gsstsig.md:60]: Oops, it's correct behind the patch fold.

File: _includes/manuals/1.10/4.3.5.3_gsstsig.md [hunk 3]
```diff
@@ -32,8 +33,21 @@ Once the keytab file has been created, test it using kinit:
 If this works, clear the Kerberos ticket cache once again using `kdestroy`.
 
 Store the keytab at `/etc/foreman-proxy/dns.keytab`, ensure permissions are 0600 and the owner is `foreman-proxy`.
+If you are using SELinux, do not forget to update the file context.
 
-The DNS zone `Dynamic Updates` option on the DNS zones can now be set to `Secure Only`. Now follow the steps below under `Proxy Configuration`.
+Now update the configuration in /etc/foreman-proxy/settings.d/dns.yml for the smart proxy:
```
Comment (domcleal) [inline _includes/manuals/1.10/4.3.5.3_gsstsig.md]: This is already largely in the proxy configuration part, should we avoid adding this paragraph?  I don't know how clear it is that the proxy config section applies to both AD and FreeIPA.

Comment (mmoll) [inline _includes/manuals/1.10/4.3.5.3_gsstsig.md]: indeed

**Comment by mmoll** [comment]
updated.

**Comment by domcleal** [comment]
Thanks for reviving this!

**Pull request merged**